spec/controllers/api/v0x0/application_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+RSpec.describe ApplicationController, :type => :request do
```

Comment:
`spec/controllers/api/v0x0/application_controller_spec.rb`

@gtanzillo should this be in `spec/controllers/application_controller_spec.rb` instead of under `api/v0x0` ?

Reply:
Yes, I'll move it